BrightID/src/actions/apps.ts

```diff
@@ -142,7 +142,7 @@ export const updateBlindSig =
           sigInfo.uid
         ) {
           console.log('removing sig and retrying');
-          dispatch(removeSig(sigInfo.uid));
+          await dispatch(removeSig(sigInfo.uid));
           await dispatch(updateBlindSig(app));
```

Comment:
Here the await makes sense, because `updateBlindSig(app)` is an AppThunk that returns a promise.